### DISCUSSION TITLE
Предложение для openvk.su: добавить "дизлайки"

### DISCUSSION THREAD

**Discussion by m65v19** [OpenVK/openvk#33] [Ideas] [answered]
В оригинальном ВК часто можно было видеть комментарии, мол дизлаков не хватает (в первую очередь для постов) Были разработы различные плагины и расширения, добавляющию данную фичу. Однако все они бесполезны по различным причинам, будь то локальность данных, отстутсвие общей базы данных, не универсальность и малое число пользователей. 
В силу вышеописсаного, предлагаю добавить в новый прекрасный OpenVK эту фичу.

**Comment by ghost** [comment, marked as answer]
Сделаем. Это будет необязательной фичей (которую можно включить в конфиге)
И этот запрос является дубликатом #27

**Comment by NoPlagiarism** [comment]
Не планируется. Можно закрывать xD
